documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -632,6 +632,28 @@ This value will be doubled upon every retry but won't exceed `debezium.sink.redi
 |`10000`
 |Max delay when encountering Redis connection or OOM issues.
 
+|[[redis-trim-maxlen]]<<redis-trim-maxlen, `debezium.sink.redis.trim.maxlen`>>
+|
+|(Optional) Evicts entries as long as the stream's length exceeds the specified threshold.
+
+|[[redis-trim-relative-minid]]<<redis-trim-relative-minid, `debezium.sink.redis.trim.relative.minid`>>
+|
+|(Optional) Specify the milliseconds, relative to the current time,
```

Comment:
Is this the right description? "minid" doesn't seem to indicate some time or interval but rather an id. If it actually is a time interval (which I don't think, based on a quick look into Redis docs), then the property name should end with ".ms" by convention.

Reply:
It is both, ids are basically time stamps in ms.If N messages have the ms, the ID will take this format. ‘ms-N‘.

Let me make the change